server.js

```diff
@@ -100,9 +100,12 @@ const globalVariables = {
         udp: false,
         web: false,
         system: false
-    }
+    },
+    useHTTPS: !(isLightweightMobile || isStandaloneMobile) // on mobile devices node.js doesn't fully support HTTPS
```

Comment:
Thinking through the workflow if we ever want to get a customer to run the server for a local access usecase (something that one of them just requested) it would be better for the server to gracefully fall back to http if the certs aren't available instead of forcing https on for all desktop users. This would take the form of checking whether cert.pem/key.pem exists, warning if not on mobile but still setting useHTTPS to false.

Reply:
My hope is that https will be come available on node.js for ios and unify the server to always run in a secure context (this solves problems with [features that want https](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts/features_restricted_to_secure_contexts)). if we enable this fallback that will make this unification much harder to do later. @Benolds What do you think?

Reply:
Hmm, I guess I'm convinced. I think I'm okay with targeting HTTPS everywhere and the real target is documentation and messaging around certificate generation and trusting such that it's not a massive pain